sample/F8/tracks

```diff
@@ -0,0 +1,202 @@
+[
+   {
+      "id":1,
+      "name":"ADS & MONETIZATION",
+      "color":"#BF532E",
+      "description":null,
+      "location":"C",
```

Comment:
The description too @niranjan94 ? In that case, should I close the issue mentioned above?